README.md

```diff
@@ -4,11 +4,10 @@ Cabot
 
 Cabot is a free, open-source, self-hosted infrastructure monitoring platform that provides some of the best features of [PagerDuty](http://www.pagerduty.com), [Server Density](http://www.serverdensity.com), [Pingdom](http://www.pingdom.com) and [Nagios](http://www.nagios.org) without their cost and complexity. (Nagios, I'm mainly looking at you.)
 
-It provides a web interface that allows you to monitor services (e.g. "Stage Redis server", "Production ElasticSearch cluster") and send telephone, sms or hipchat/email alerts to your on-duty team if those services start misbehaving or go down - all without writing a line of code. Best of all, you can use data that you're already pushing to Graphite/statsd to generate alerts, rather than implementing and maintaining a whole new system of data collectors.
+It provides a web interface that allows you to monitor services (e.g. "Stage Redis server", "Production ElasticSearch cluster") and send telephone, sms or hipchat/email alerts to your on-duty team if those services start misbehaving or go down - all without writing a line of code. Best of all, you can use data that you're already pushing to statsd to generate alerts, rather than implementing and maintaining a whole new system of data collectors.
 
 You can alert based on:
 
-*   Metrics from [Graphite](https://github.com/graphite-project/graphite-web)
 *   Status code and response content of web endpoints
 *   [Jenkins](http://jenkins-ci.org) build statuses
```

Comment:
We should update this sometime (add ES/TCP), change the intro, etc. Doesn't have to be in this PR tho